etc/adminhtml/system.xml

```diff
@@ -122,6 +122,11 @@
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                     <comment>Enable view transitions when loading pages on the same domain.</comment>
                 </field>
+                <field id="enable_admin" translate="label tooltip" type="select" sortOrder="12" showInDefault="1" showInWebsite="1" showInStore="1" canRestore="1">
+                    <label>Enable View Transitions for Admin</label>
+                    <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
+                    <tooltip>Don't enable this when using the Mage-OS Admin Theme or any other Admin theme that supports View Transitions</tooltip>
```

Comment:
Wording change, but

1. Is it harmful to have it enabled twice, other than small redundant script per page?;
2. Given it's explicitly per-theme, surely this could be handled better than 'set to no if (XYZ system state)'. Could we have the admin themes use this module for it instead? Or something else?

```suggestion
                    <tooltip>Disable if you don't want the view transitions, or when using the Mage-OS Admin Theme or any other Admin theme that has View Transitions built in</tooltip>
```